src/com/example/ui.cljs

```diff
@@ -131,29 +131,56 @@
                  (p :.max-w-prose.mb-5.px-4
                     "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.")))))
 
+(defsc Dashboard [this {:keys [readiness-quiz] :root/keys [response]}]
+  {:route-segment ["dashboard"]
+   :query [{:readiness-quiz [:quiz/id]}
+           {[:root/response '_] [:response/total]}]
```

Comment:
this is a "Link Query" ([doc](https://book.fulcrologic.com/#_link_queries) It allows us to query the root of the database. Fulcro will usually walk from the relative edge in the data graph.